data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/special_damage_type.cfg

```diff
@@ -348,3 +348,160 @@
         {SUCCEED}
     [/event]
 )}
+
+#####
+# API(s) being tested: [damage]alternative_type=
+##
+# Actions:
+# Give both Alice and Bob 100% chance to hit.
+# Give Bob one [damage] with alternative_type=cold
+# change resistance Bob to fire to -100% in using [resistance] ability.
+# Give Alice one [damage] with alternative_type=fire
+# and change resistance to cold to -100% and blade to 0%.
```

Comment:
If the specials of Alice and Bob are doing different things, then this should be two tests instead of mixing them together.

That applies even if it's testing "works on offense" and "works on defense"; however it that case it might be better to have a single test that does:
* alice attacks bob
* check result
* bob attacks alice
* check result